Gemfile

```diff
@@ -67,14 +73,16 @@ gem "clamav"
 gem "pdf-reader"
 
 # to generate sitemap for google scholar et al
-gem 'sitemap', github: 'ualbertalib/rails-sitemap'
+gem 'sitemap', git: 'https://github.com/ualbertalib/rails-sitemap.git'
 
 # to fetch noid from fedora for reindex job
 gem 'rest-client'
 
 gem 'noid', '~> 0.8'
 
 group :test do
+  gem 'vcr', require: false
```

Comment:
I'd not heard about vcr before. Is there anything to keep back of mind when using it?